ui/src/app/components/buttons.tsx

```diff
@@ -325,6 +326,41 @@ export const ButtonWithLocationState = ({
   );
 };
 
+// uses HashLink to allow navigation to anchor IDs on the same page
+
+// example: if you have a component with id='user-info', then you can create a HashLinkButton which navigates to it:
+// <HashLinkButton path='#user-info'/>
+
+export const HashLinkButton = ({
```

Comment:
a copy of our `Button` which uses `react-router-hash-link`.  Open to alternatives.